elements/lisk-chain/src/block_reward.ts

```diff
@@ -121,3 +125,94 @@ export const calculateSupply = (
 
 	return supply;
 };
+
+export const getTotalRewardAndFees = (
+	blockInstance: BlockInstance,
+): { readonly totalFee: bigint; readonly totalBurnt: bigint } => {
+	// tslint:disable-next-line no-let
+	let totalFee = BigInt(0);
+	// tslint:disable-next-line no-let
+	let totalBurnt = BigInt(0);
+
+	for (const transaction of blockInstance.transactions) {
+		const minFee = BigInt(0); // TODO: It should get from transaction
```

Comment:
We do have an issue for this task, can we add a link to that issue directly in the comment.